core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestrator.scala

```diff
@@ -93,14 +92,12 @@ class BillingProjectOrchestrator(ctx: RawlsRequestContext,
         creationStatus,
         None
       )
-    } yield {}).recover {
-      case e: RawlsExceptionWithErrorReport =>
-        e.errorReport.statusCode.collect {
-          case _: ServerError =>
-            tagAndCaptureSentryEvent(e)
-          case _ => throw e
-        }
-      case wsmException: WsmApiException if wsmException.getCode >= 500 => tagAndCaptureSentryEvent(wsmException)
+    } yield {}).recover { case e: RawlsExceptionWithErrorReport =>
+      e.errorReport.statusCode.collect {
+        case _: ServerError =>
+          tagAndCaptureSentryEvent(e)
+        case _ => throw e
+      }
```

Comment:
removes the `case wsmException: WsmApiException` case, which will never happen now that we are not making API calls to WSM.